diagnostic.md

```diff
@@ -32,7 +41,12 @@ y = 10
 
 After each line of code executes, what are the values of x and y?  Do the variables change?  Do their values?
 
-<!-- solution below -->
+let x = 4 --> 4
+let y = 3 --> 3
+x = y --> 3
+y = 10 --> 10
+
+The variables are change each time you assign them a different value.
```

Comment:
Close here, the variables actually remain `x` and `y` but their values do change.